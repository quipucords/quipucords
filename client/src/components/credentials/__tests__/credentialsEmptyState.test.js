import React from 'react';
import { mount } from 'enzyme';
import CredentialsEmptyState from '../credentialsEmptyState';

describe('CredentialsEmptyState Component', () => {
  it('should render a basic component', () => {
    const props = {};

    const component = mount(<CredentialsEmptyState {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
