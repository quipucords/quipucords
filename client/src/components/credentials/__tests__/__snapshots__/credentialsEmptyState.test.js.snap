// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CredentialsEmptyState Component should render a basic component 1`] = `
<div
  class="container-fluid"
>
  <div
    class="row"
  >
    <div
      class="blank-slate-pf full-page-blank-slate"
    >
      <div
        class="blank-slate-pf-icon"
      >
        <span
          aria-hidden="true"
          class="pficon pficon-add-circle-o"
        />
      </div>
      <h4
        class="h1 blank-slate-pf-title"
      >
        Welcome to Red Hat Entitlements Reporting
      </h4>
      <p
        class="blank-slate-pf-info"
      >
        A credential defines a set of user authentication information to be used during a scan.
        <br />
        A credential includes a username and a password or SSH key. Entitlements Reporting uses SSH
        <br />
        to connect to servers on the network and uses credentials to access those servers.
      </p>
      <div
        class="blank-slate-pf-main-action"
      >
        <div
          class="dropdown btn-group btn-group-lg btn-group-primary"
        >
          <button
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle btn btn-lg btn-primary"
            id="createCredentialButton"
            role="button"
            type="button"
          >
            Add Credential 
            <span
              class="caret"
            />
          </button>
          <ul
            aria-labelledby="createCredentialButton"
            class="dropdown-menu dropdown-menu-right"
            role="menu"
          >
            <li
              class=""
              role="presentation"
            >
              <a
                href="#"
                role="menuitem"
                tabindex="-1"
              >
                Network Credential
              </a>
            </li>
            <li
              class=""
              role="presentation"
            >
              <a
                href="#"
                role="menuitem"
                tabindex="-1"
              >
                Satellite Credential
              </a>
            </li>
            <li
              class=""
              role="presentation"
            >
              <a
                href="#"
                role="menuitem"
                tabindex="-1"
              >
                VCenter Credential
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        class="blank-slate-pf-secondary-action"
      >
        <button
          class="btn btn-default"
          type="button"
        >
          Add Source
        </button>
      </div>
    </div>
  </div>
</div>
`;
